src/libsystemd-network/radv-internal.h

```diff
@@ -87,6 +87,7 @@ struct sd_radv {
         uint8_t hop_limit;
         uint8_t flags;
         uint32_t mtu;
+        uint32_t retransmit;
```

Comment:
Please add a short comment that this is in milliseconds. or name `retransmit_msec`.
Otherwise we need to refer RFC 4861 to find its unit.